promptify/models/nlp/openai_model.py

```diff
@@ -15,13 +24,13 @@ def __init__(self, api_key: str, model: str = "text-davinci-003"):
         self.model = model
         self._openai = openai
         self._openai.api_key = self._api_key
-        self.supported_models = [
+        self.supported_models = (
```

Comment:
Default arguments should not use mutable types, so switched from `list` to `tuple`

Reply:
agreed, good that you changed it